python/py-paho-mqtt/Portfile

```diff
@@ -4,23 +4,22 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-paho-mqtt
-version             1.5.1
+version             1.6.1
 revision            0
 
-# Eclipse Public License v1.0 / Eclipse Distribution License v1.0
-license             unknown
+license             {EPL-2 EDL-1}
 maintainers         {mojca @mojca} openmaintainer
 
 description         MQTT version 3.1/3.1.1 client class
 long_description    Eclipse Paho MQTT Python client library
 
 homepage            https://eclipse.org/paho
 
-checksums           rmd160  16be65b419a07bfa3584881b7dd75e2332ebd534 \
-                    sha256  9feb068e822be7b3a116324e01fb6028eb1d66412bf98595ae72698965cb1cae \
-                    size    101757
+checksums           rmd160  d6bccf7fdb166109d901a63fafc99432c195ef68 \
+                    sha256  2a8291c81623aec00372b5a85558a372c747cbca8e9934dfe218638b8eefc26f \
+                    size    99373
 
-python.versions     37 38 39
+python.versions     37 38 39 310
```

Comment:
Are dependencies not supporting `311` yet?

Reply:
Upstream says nothing about 311 support: https://github.com/eclipse/paho.mqtt.python/blob/9782ab81fe7ee3a05e74c7f3e1d03d5611ea4be4/setup.py#L54